webpack/scenes/Subscriptions/SubscriptionReducer.js

```diff
@@ -100,7 +90,7 @@ export default (state = initialState, action) => {
     case SUBSCRIPTIONS_QUANTITIES_SUCCESS: {
       return state.merge({
         quantitiesLoading: false,
-        availableQuantities: mapQuantities(action.response.results),
```

Comment:
`mapQuantities` moved to the helper and getting used in the action so we get the actual results into the payload object.